packages/kmod-5.15-nvidia/nvidia-persistenced.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=NVIDIA Persistence Daemon
+Requires=load-kernel-modules.service
+After=load-kernel-modules.service
+
+[Service]
+Type=forking
+# Run the NVIDIA System Management Interface to create the device nodes.
+ExecStart=__NVIDIA_BINDIR__/nvidia-persistenced
```

Comment:
In 5.15 and 6.1, the path to `nvidia-persistenced` is more consistent and deterministic (it doesn't include the version). It should be fine to just use the full path here and you will save yourself the extra `sed`.